static/components/AroundCards.js

```diff
@@ -37,7 +37,7 @@ class App extends React.Component {
           userPosition = [position.coords.longitude, position.coords.latitude]
           focusUser(userPosition)
           this.setState({
-            userPosition: [userPosition[1], userPosition[0]]
+            userPosition: [userPosition[0], userPosition[1]]
```

Comment:
I want change because your currentLocation is LngLat and I would like a consistent code